### PR TITLE
[4.0] api missing import

### PR DESCRIPTION
**Pull request by alikon**
~~Pull Request for Issue #26846~~

### Summary of Changes
added missed import of `use Joomla\CMS\Language\Text;`

### Testing Instructions
~~see #26846 or~~
code review

### CODE REVIEW

File: api/components/com_contact/Controller/ContactController.php [hunk 1]
```diff
@@ -14,6 +14,7 @@
 use Joomla\CMS\Component\ComponentHelper;
 use Joomla\CMS\Factory;
 use Joomla\CMS\Form\Form;
+use Joomla\CMS\Language\Text;
```
Comment (Quy) [inline api/components/com_contact/Controller/ContactController.php:17]: Duplicate. Delete line 26.

Comment (alikon) [reply api/components/com_contact/Controller/ContactController.php:17]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on bc53c76aca5dfc3869f0bdcd61a39a09ce5017a3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26864">issues.joomla.org/tracker/joomla-cms/26864</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on bc53c76aca5dfc3869f0bdcd61a39a09ce5017a3<br /><br />I am still getting

            "title": "Save failed with the following error: COM_CONTACT_ERROR_UNIQUE_ALIAS"
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26864">issues.joomla.org/tracker/joomla-cms/26864</a>.</sub>

**Comment by alikon** [comment]
you are right it doesn't  solve #26846 .....i'll reopen it
anyway this pr have a sense filling the missing library

**Comment by brianteeman** [comment]
the problem is not with the use statement. In joomla we had 2 applications and therefore we had a language folder in each application (site and administrator). We now have an additional application (api) and so we should really have an application language folder. It is not enough to _proxy_ to either the site or the administrator folder as the strings could be in either.

**Comment by alikon** [comment]
don't forget the CLI too :smile:

**Comment by alikon** [comment]
i've raised this "general" problem on the jbs glip channel, but unfortunately you are not in there....grrr
we have got some feedback..... about reverting #12124 as a quick fix....

**Comment by brianteeman** [comment]
I doubt very much that #12124 will have any impact on this at all

**Comment by alikon** [comment]
well, right now i dunno, i need to check, but the "one" who proposed this quick fix, was the "one who wrote that pr ...so i'm inclined to give it a try :100:

**Comment by wilsonge** [comment]
That PR isn't the fix. It's what @brianteeman says (I tested this with him in the pub on Tuesday night). I need to dig around in the code to actually prove why though

**Comment by wilsonge** [comment]
Clearly imports are missing here though - so merging this! Thanks!

**Pull request merged**